README.md

```diff
@@ -81,72 +81,57 @@ but as necessary, rebuild the image using Step 1.
-
-16. Add all scopes needed by the application, i.e. so that they match those listed in `ccm_web/server/src/canvas/canvas.scopes.ts`.
-
-17. Click the "Save" button.
-
-18. Copy the ID number of the API key created in Canvas. The ID is the long number shown in the "Details" column of the "Developer Keys" page. It usually looks like `17700000000000nnn`.
-
-19. Click the "Show Key" button underneath the ID located in step 17, and copy the secret that appears in the dialog.
-
-20. Click the "ON" part of the switch in the "State" column of your API key, so that it has a green background.
-
-21. Edit the `.env` file.
-
-    1. Verify the `LTI_PLATFORM_URL` variable value is correct for the instance of Canvas used.
-    2. Add to the `LTI_CLIENT_ID` variable value the LTI key ID number copied earlier (in step 12).
-    3. Add to the `CANVAS_API_CLIENT_ID` variable value the API key copied earlier (in step 18).
-    4. Add to the `CANVAS_API_SECRET` variable value the secret copied earlier (in step 19).
-
-22. Build and start the application with docker-compose.
+10. Copy the ID number of the LTI key created in Canvas.  The ID is the long number shown in the "Details" column of the "Developer Keys" page.  It usually looks like "`17700000000000nnn`".
+11. Click the "ON" part of the switch in the "State" column of your LTI key, so that it has a green background.
+12. Go to Canvas "Developer Keys" management available from the "Admin" page for your account. Click the button for "+ Developer Key", then "+ API Key".
+13. Enter a name in the "Key Name" field, and under "Redirect URIs" enter:
+    `https://{ccm_app_hostname}/canvas/returnFromOAuth`
+    Where `{ccm_app_hostname}` is the ngrok hostname copied earlier (in step 2).
+    ***Note:*** Do ***NOT*** use the "Redirect URI (Legacy)" field.
+14. Enable the "Enforce Scopes" option, then add all scopes needed by the application (i.e., those listed in `ccm_web/server/src/canvas/canvas.scopes.ts`).
+15. Click the "Save" button.
+16. Copy the ID number of the API key created in Canvas. The ID is the long number shown in the "Details" column of the "Developer Keys" page. It usually looks like "`17700000000000nnn`".
+17. Click the "Show Key" button underneath the ID located in step 15, and copy the secret that appears in the dialog.
+18. Click the "ON" part of the switch in the "State" column of your API key, so that it has a green background.
+19. Make a `.env` file for the project, based on the sample provided.
+    `cp config/.env.sample .env`
+20. Edit the `.env` file.  The keys in the following list must/should be updated.
+
+    1. `DOMAIN` – Hostname of the server hosting the CCM application.
+       For local development purposes with ngrok, use the hostname
+       copied during step 2.
+    2. Verify the `LTI_PLATFORM_URL` variable value is correct for the instance of Canvas used.
+    3. Add to the `LTI_CLIENT_ID` variable value the LTI key ID number copied earlier (in step 10).
```

Comment:
At some point we need a better way of generating the `LTI_ENCRYPTION_KEY`. Currently it is defaulting to `LTIKEY` which is good for local. When we have to think of Non-prod and Prod env we should figure out how these keys are can be generated. May be Base 64 encode stuff

Same things goes with `DOMAIN`.

I don't think this PR need to address that, but I feel once we understand how to generate those keys vales README would be a place to document. Perhaps separate issue?